lib/gooddata/models/process.rb

```diff
@@ -209,6 +211,33 @@ def deploy_from_appstore(path, options = { :client => GoodData.client, :project
         process
       end
 
+      def deploy_component(data, options = { :client => GoodData.client, :project => GoodData.project })
+        client, project = GoodData.get_client_and_project(options)
+        process_id = data[:process_id]
+        data = { process: data } unless data[:process]
+        data[:process ] = GoodData::Helpers.symbolize_keys(data[:process]).select { |k| [:type, :name, :component].include? k }
+        data[:process][:component] = GoodData::Helpers.symbolize_keys(data[:process][:component]).select { |k| [:name, :version].include? k }
+
+        # TODO: we should not have to do this
+        client.post "/gdc/projects/#{project.pid}/projectFeatureFlags",
+                    featureFlag: {
+                      key: 'enableEtlComponent',
+                      value: true
+                    }
+
+        res =
```

Comment:
Identical code found in process.rb:122, process.rb:196, process.rb:228 (mass = 198)